cartography/intel/aws/ec2.py

```diff
@@ -72,6 +78,35 @@ def get_ec2_vpcs(session, region):
     return client.describe_vpcs()
 
 
+def load_ec2_key_pairs(session, data, region, current_aws_account_id, aws_update_tag):
+    ingest_key_pair = """
+    MERGE (keypair:KeyPair:EC2KeyPair{arn: {ARN}})
```

Comment:
Change this to 

`MERGE (keypair:KeyPair:EC2KeyPair{arn: {ARN}, id: {ARN}})`.

We're trying to use `id` as a universal unique identifier for all node types (even though there are some inconsistencies in other parts of the codebase :\).  In this case, an `id` is the same value as the `arn`.

Reply:
Changed.